source/core/Variants.cpp

```diff
@@ -167,16 +166,15 @@ VIREO_FUNCTION_SIGNATURET(GetVariantAttribute, GetVariantAttributeParamBlock)
                     found = true;
                     value->_paramType->CopyData(foundValue->Begin(kPARead), value->_pData);
                 } else {
-                    if (errPtr) {
-                        errPtr->SetErrorAndAppendCallChain(true, 91, "Get Variant Attribute");  // Incorrect type for default value for the attribute
+                    if (errPtr) {  // Incorrect type for default attribute value
```

Comment:
This is fine here again. Please add to the same github issue to define these as string constants and consolidate. Duplicating it just asks for misspelling and causing error call chain to fail in tests.

Reply:
Good point. Updated the following issue: https://github.com/ni/VireoSDK/issues/384